packages/web/src/app/App.tsx

```diff
@@ -19,8 +19,7 @@ import WebPlayer from './web-player/WebPlayer'
 import '../services/webVitals'
 
 const SignOn = lazy(() => import('pages/sign-on/SignOn'))
-const SignInPage = lazy(() => import('pages/sign-in-page'))
-const SignUpPage = lazy(() => import('pages/sign-up-page'))
+const SignOnPage = lazy(() => import('pages/sign-on-page'))
```

Comment:
the naming here feels asymmetric now

Reply:
im down to make things more explicit with "signonlegacy"? it just happens to work with "sign-on" vs "sign-on-page" lol